WSL/wsl2-mount-disk.md

```diff
@@ -9,9 +9,9 @@ ms.localizationpriority: medium
 
 # Get started mounting a linux disk in WSL 2 (preview)
 
-If you're dual booting to Linux, or simply want to access a linux disk format that isn't supported by Windows, WSL 2 can mount your disk and let you access its content.
+If you want to access a Linux disk format that isn't supported by Windows, you can use WSL 2 to mount your disk and access its content. No more need to dual-boot.
```

Comment:
"No more need to dual-boot." -- Now this is called monopoly.

Reply:
? How so? It's removing the restriction of having to boot into two different systems and saving a lot of time. It also doesn't eliminate anyone's ability to dual-boot. Not sure what the beef is here...?

Reply:
That line conveys the message that -- No one need to dual boot. It's not the 'ability', it's mindset.

Reply:
Seems a bit semantic to me. Do you have an alternative to suggest?

Reply:
In my opinion, just remove the last line.

Reply:
Hmm, not opposed to that. Thanks. I'll talk it over with the team.